src/overmapbuffer.cpp

```diff
@@ -298,15 +304,16 @@ std::vector<om_vehicle> overmapbuffer::get_vehicle(int x, int y, int z, bool req
     return result;
 }
 
-void overmapbuffer::signal_hordes( const tripoint center, const int sig_power )
+void overmapbuffer::signal_hordes( const tripoint &center, const int sig_power )
 {
     const auto radius = sig_power;
     for( auto &om : get_overmaps_near( center, radius ) ) {
         const point abs_pos_om = om_to_sm_copy( om->pos() );
-        const point rel_pos( center.x - abs_pos_om.x, center.y - abs_pos_om.y );
+        // TODO: Z
```

Comment:
What's missing here? The usage of `center.z` seems correct, it's an absolute value, so it doesn ot need to be translated to relative coordinates (relative to the location of the overmap).